examples/numpy_to_strain.py

```diff
@@ -0,0 +1,50 @@
+# Demonstrate loading numpy displacement data and
+# generating strain output with ITK Python
+
+import itk
+import numpy as np
+
+assert 'StrainImageFilter' in dir(itk)
+
+def create_numpy_displacement() -> np.ndarray:
+	# Create 100x100 image of increasing displacements with x/y components
```

Comment:
Aren't displacements random?

Reply:
You're right, comment was left over from an earlier attempt whose output didn't do a good job of illustrating strain. Amended.